docs/about-block-inline-tags.md

```diff
@@ -7,8 +7,8 @@ description: Overview of block and inline JSDoc tags.
 
```

Comment:
Note: Have done the following changes in the markdowns -
1. Replaced `{% example "Inline tag used within a block tag" %}` with `* **Inline tag used within a block tag**`.
2. Removed `{% endexample %}`.
3. Removed `.html` suffix from internal links.
4. Prettier stuff